pkg/kvstore/consul.go

```diff
@@ -483,12 +493,15 @@ func (c *consulClient) Delete(ctx context.Context, key string) error {
 }
 
 // GetIfLocked returns value of key if the client is still holding the given lock.
-func (c *consulClient) GetIfLocked(ctx context.Context, key string, lock KVLocker) ([]byte, error) {
+func (c *consulClient) GetIfLocked(ctx context.Context, key string, lock KVLocker) (bv []byte, err error) {
+	defer func() { Trace("GetIfLocked", err, logrus.Fields{fieldKey: key, fieldValue: string(bv)}) }()
 	return c.Get(ctx, key)
 }
 
 // Get returns value of key
-func (c *consulClient) Get(ctx context.Context, key string) ([]byte, error) {
+func (c *consulClient) Get(ctx context.Context, key string) (bv []byte, err error) {
+	defer func() { Trace("Get", err, logrus.Fields{fieldKey: key, fieldValue: string(bv)}) }()
+
 	duration := spanstat.Start()
 	qo := &consulAPI.QueryOptions{}
 	pair, _, err := c.KV().Get(key, qo.WithContext(ctx))
```

Comment:
`s/:=/=/`

Reply:
`pair` is a new variable, though. If you're thinking that err won't be set correctly, I think that's only an issue with ifs (my test fmt.Println("Hello, playground", foo()))